src/comment-handlers/line-comments-handler.ts

```diff
@@ -22,6 +22,65 @@ function createCommentMarkerId(path: string, line: number): string {
   return `${path}:${line}`.replace(/[^a-zA-Z0-9-_:.]/g, '_')
 }
 
+/**
+ * Extracts the marker ID from a comment body
+ */
+function extractMarkerIdFromComment(commentBody: string): string | null {
+  const match = commentBody.match(
+    new RegExp(
+      `${COMMENT_MARKER_PREFIX.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')}(.+?)${COMMENT_MARKER_SUFFIX.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')}`
+    )
+  )
+  return match ? match[1] : null
+}
+
+/**
+ * Cleans up obsolete comments that are no longer relevant in the current diff
+ */
+async function cleanupObsoleteComments(
+  context: Context,
+  prNumber: number,
+  diffMap: Map<string, { changedLines: Set<number> }>,
+  existingComments: Array<{ id: number; body: string; path?: string }>
+): Promise<number> {
```

Comment:
Visiblement, on n'utilise pas `prNumber`
```suggestion
async function cleanupObsoleteComments(
  context: Context,
  diffMap: Map<string, { changedLines: Set<number> }>,
  existingComments: Array<{ id: number; body: string; path?: string }>
): Promise<number> {
```